packages/react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -111,6 +111,37 @@ function useBounds(
   return getBounds;
 }
 
+/**
+ * (Hook) to return the maximum available height for the Callout to render into.
+ */
+function useMaxHeight(
+  { calloutMaxHeight, directionalHint, directionalHintFixed, hidden }: ICalloutProps,
+  positions?: ICalloutPositionedInfo,
+  getBounds?: () => IRectangle,
+) {
+  const [maxHeight, setMaxHeight] = React.useState<number | undefined>();
+  const { top, bottom } = positions?.elementPosition ?? {};
+
+  React.useEffect(() => {
+    const isForcedInBounds = typeof top === 'number' && typeof bottom === 'number';
+    const { top: topBounds, bottom: bottomBounds } = getBounds ? getBounds() : ({} as IRectangle);
```

Comment:
Probably should revert this to go with the change above:

```suggestion
    const { top: topBounds, bottom: bottomBounds } = getBounds() : ({} as IRectangle);
```